app/src/main/java/unipd/se18/ocrcamera/Inci.java

```diff
@@ -0,0 +1,148 @@
+        listIngredientFound = new ArrayList<String>();
+        loadDB(inputStream);
+    }
+
+    public ArrayList<String> getListInci(){
+        return this.listInci;
+    }
+
+    public ArrayList<String> getListInciDescription(){
+        return this.listInciDescription;
+    }
+
+    public ArrayList<String> getListIngredientFound(){
+        return this.listIngredientFound;
+    }
+
+
+    /**
+     * @author Giovanni Fasan(g1)
+     * @param inputStream of the R.raw.database
+     * @return void
+     * Load every Inci ingredients to arrayList
+     */
+    private void loadDB(InputStream inputStream){
+        //define 0 the name of the ingredients
+        int name=0;
+        //define 1 the description of the ingredients
+        int description=1;
+        //open Buffer Reader for read the database
+        BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream, Charset.forName("UTF-8")));
+        String ingredient;
+        try {
+            //slide every line of the database and add in the appropriate list
+            while ((ingredient = reader.readLine()) != null) {
+                String[] inci=ingredient.split(";");
+                listInci.add(inci[name].trim());
+                //TODO listInciDescription
+                //listInciDescription.add(inci[description].trim());
+            }
+            Log.d("inci", "create listInci");
+        }catch (Exception e){
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * @author Giovanni Fasan(g1)
+     * @return String with ingredients list of the label
+     * Concatenate all the ingredients found in a string. It puts the ingredidients in bold
+     */
+
+    public String ingredientsFoundToString() {
+        String inci="";
+        for (int j=0; j<this.listIngredientFound.size(); j++){
+            inci = inci + "<b>" + listIngredientFound.get(j) + "</b>; ";
+        }
+
+        return inci;
+    }
+
+    /**
+     * @author Giovanni Fasan(g1)
+     * @param text String in which you have to find the ingredients
+     * loads all the ingredients inci found in the ArrayList listIngredientFound
+     */
+    private void findIngredientsList_old(String text) {
+        listIngredientFound = new ArrayList<String>();
+        //remove every \n
+        //split by Ingredients to reduce the number of words to be analyzed
+        String noEnd = text.replaceAll("\n", " ");
+        String [] word = noEnd.split(",");
+        for (int i = 0; i < word.length; i++) {
+            for (int j=0; j<this.listInci.size(); j++){
+                if (word[i].trim().toUpperCase().equals(listInci.get(j))) {
+                    listIngredientFound.add(listInci.get(j));
+                }
+            }
+        }
+
+    }
+
+
+    /**
+     * @author Giovanni Fasan(g1)
+     * @param text String in which you have to find the ingredients
+     * loads all the ingredients inci found in the ArrayList listIngredientFound
```

Comment:
In order to comply javadocs you should put the description first, followed by block tags. (see "How to Write Doc Comments for the Javadoc Tool"  https://www.oracle.com/technetwork/java/javase/index-137868.html)